utils/helpers.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Returns a random object from the array of colors to user
+ * @param array {array} : array containing objects
+ * @returns random Index number : index between the range 0 to array.length
+ */
+const getRandomIndex = (arrayLength = 10) => {
```

Comment:
Nit: we call the function parameter `maxIndex` so that parameter represents the maximum index value that the random index should not exceed.

Reply:
updated